docs/en_US/oauth2.rst

```diff
@@ -39,6 +43,10 @@ and modify the values for the following parameters:
     "OAUTH2_AUTO_CREATE_USER", "Set the value to *True* if you want to automatically
     create a pgAdmin user corresponding to a successfully authenticated Oauth2 user.
     Please note that password is not stored in the pgAdmin database."
+    "OAUTH2_ADDITIONAL_CLAIMS", "The additional claims to check on the openid ID token. 
+    If the value is provided, and the user ID Token has any of the given values, the login flow will continue.
```

Comment:
The description is not very clear.

Reply:
I'm willing to change but i'm out of ideas, how would it look very clear to you?

Reply:
for reference, this is how hashicorp vault implement a similar feature:

![image](https://github.com/pgadmin-org/pgadmin4/assets/30481051/59759472-5c3a-4269-a663-af169da2ae60)

Reply:
changed, can you see if it looks better now?